feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.spec.ts

```diff
@@ -259,20 +259,22 @@ describe('ConfiguratorCartEntryBundleInfoComponent', () => {
 
   describe('isBundleBasedConfigurator', () => {
     it('should return false because the configurator type is not bundle based one', () => {
+      let testEntry = structuredClone(entry);
```

Comment:
What do you think about renaming the variable to `modified`- or `clonedEntry`? The same for the next two tests. :)